roles/openshift_logging_mux/tasks/main.yaml

```diff
@@ -177,6 +177,48 @@
   check_mode: no
   changed_when: no
 
+# storageclasses are used by default but if static then disable
+# storageclasses with the storageClassName set to "" in pvc.j2
```

Comment:
Ok.  It makes sense.  Let me move them to the common place (is openshift_logging ok?) to share...